database/migrations/2017_03_21_042011_CreateExpensesTable.php

```diff
@@ -0,0 +1,37 @@
+<?php
```

Comment:
Is this different than the one we have in Staging? We already have this kind of migration fie